sources/packages/web/src/views/student/financial-aid-application/FullTimeApplication.vue

```diff
@@ -69,18 +102,19 @@ import {
 } from "@/types";
 import { ApplicationDataAPIOutDTO } from "@/services/http/dto";
 import { StudentRoutesConst } from "@/constants/routes/RouteConstants";
-import ConfirmEditApplication from "@/components/students/modals/ConfirmEditApplication.vue";
 import {
   STUDY_DATE_OVERLAP_ERROR,
   ACTIVE_STUDENT_RESTRICTION,
 } from "@/constants";
 import StudentApplication from "@/components/common/StudentApplication.vue";
 import { AppConfigService } from "@/services/AppConfigService";
+import ConfirmModal from "@/components/common/modals/ConfirmModal.vue";
+import { BannerTypes } from "@/types";
```

Comment:
Please move the BannerTypes import here. Sonar cloud was complaining that.

![image](https://github.com/user-attachments/assets/2231d6f0-cbb2-4910-8cd5-db10a0bb8da6)